manim/mobject/text/text_mobject.py

```diff
@@ -147,47 +147,29 @@ class Paragraph(VGroup):
     def __init__(self, *text, line_spacing=-1, alignment=None, **config):
```

Comment:
I don't like that the additional keyword arguments are named `config` here, I'd rather see them be named `kwargs` as everywhere else. This is more venting than anything else though, not a change request. :-)

Reply:
Huh, for a moment I thought that `config` was used more often. I now checked and you're right, pretty much everywhere else they're named `kwargs`. To be more consistent, yeah, I would change it to `kwargs`.

Reply:
I just noticed that using `config` as a name shadows the `config` global variable, so this is actually an issue which should be solved. I'd definitely change it to `kwargs`.